.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: xenial
 language: elixir
 elixir:
-  - 1.6.4
+  - 1.6.6
```

Comment:
Do we actually want to make the elixir version to 1.6.6 with this PR?

Reply:
@cytadela8 I assume we can't support 1.6.6 yet, as it is for erlang 21, where the rox library doesn't work yet, or did you find a way to make it work?

https://github.com/urbint/rox/pull/15

Reply:
I have problem compiling older version and I think mix format behaviour changed so Travis complained on format. 1.6.6 is the up-to-date version.

Reply:
@thepiwo Just seen your comment. I have no problem compiling rox on 1.6.6 and I have OTP-21, but Travis has OTP-20 and it also works.

Reply:
did you try removing and refetching dependencies? I had some problems with this

Reply:
Hmm, the OTP difference is probably the reason why Elixir 1.6.4 doesn't compile for me.

Reply:
Ok, so I will downgrade OTP locally, compile 1.6.4 and revert 19747bc